### PR TITLE
Bump InfluxDB version

### PR DESCRIPTION
**Pull request by aadams19**

### CODE REVIEW

File: monasca/values.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ influxdb:
   enabled: true
   image:
     repo: "influxdb"
-    tag: "1.1"
```
Comment (mhoppal) [inline monasca/values.yaml]: should we use 1.3.5-alpine

**Comment by mhoppal** [comment]
+ export OS_USERNAME=mini-mon
+ export OS_PASSWORD=password
+ export OS_TENANT_NAME=mini-mon
+ export OS_DOMAIN_NAME=Default
+ export OS_AUTH_URL=http://monasca-keystone:35357/v3
+ export MONASCA_URL=http://monasca-api:8070
+ ip route get 8.8.8.8
+ awk NR==1 {print $NF}
+ export WEBHOOK_IP=172.17.0.20
+ /smoke-test
TEST MEASUREMENTS FLOWING
SUCCESS

TEST NOTIFICATION CREATION
SUCCESS

TEST ALARM DEFINITION CREATION
SUCCESS

TEST METRIC CREATION
SUCCESS

TEST WEBHOOK TRIGGERED
SUCCESS

TEST CLEANUP
SUCCESS

All smoke tests passed successfully!!!

======
Totals
======
Ran: 252 tests in 587.0000 sec.
 - Passed: 252
 - Skipped: 0
 - Expected Fail: 0
 - Unexpected Success: 0
 - Failed: 0
Sum of execute time for each test: 331.7530 sec.

==============
Worker Balance
==============
 - Worker 0 (252 tests) => 0:09:42.100839

Slowest Tests:
Test id                                                                                                                       Runtime (s)
----------------------------------------------------------------------------------------------------------------------------  -----------
monasca_tempest_tests.tests.api.test_metrics.TestMetrics.test_list_metrics_with_offset_limit[gate]                            15.791
monasca_tempest_tests.tests.api.test_metrics.TestMetrics.test_create_metric_with_colon_in_dimension_value[gate]               15.589
monasca_tempest_tests.tests.api.test_metrics.TestMetrics.test_list_metrics_dimension_query_no_value_with_diff_names[gate]     15.511
monasca_tempest_tests.tests.api.test_metrics.TestMetrics.test_list_metrics_dimension_query_multi_value_with_diff_names[gate]  15.314
monasca_tempest_tests.tests.api.test_metrics.TestMetrics.test_list_metrics_with_name[gate]                                    15.265
monasca_tempest_tests.tests.api.test_statistics.TestStatistics.test_list_statistics_with_no_merge_metrics[gate,negative]      15.017
monasca_tempest_tests.tests.api.test_metrics.TestMetrics.test_create_metrics[gate]                                            14.872
monasca_tempest_tests.tests.api.test_statistics.TestStatistics.test_list_statistics_with_offset_limit[gate]                   14.728
monasca_tempest_tests.tests.api.test_metrics.TestMetrics.test_create_metric_with_no_dimensions[gate]                          14.702
monasca_tempest_tests.tests.api.test_metrics.TestMetrics.test_list_metrics_with_project[gate]                                 14.596

File: monasca/values.yaml (hunk 1 above)
Comment (aadams19) [reply monasca/values.yaml]: Done

**Comment by mhoppal** [comment]
Both smoke and tempest tests passed!

**Comment by mhoppal** [comment]
can you bump the monasca chart version?

**Comment by aadams19** [comment]
chart bumped

**Pull request merged**